### PR TITLE
changed the color of the editor-toolbar_hover class

### PR DESCRIPTION
**Pull request by ByronEricPerez**
## Fixes

Issue Number: https://github.com/lbryio/lbry-desktop/issues/7619

## What is the current behavior?

image

## What is the new behavior?

image

## PR Checklist

<details><summary>Toggle...</summary>

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [ ] I added a line describing my change to CHANGELOG.md
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

</details>

### CODE REVIEW

**Comment by jessopb** [comment]
What do you think of something like this?
image
image
Of course adjusted to work with themes.

**Comment by ByronEricPerez** [comment]
Hi @jessopb , what do you think?

image

**Review by jessopb — COMMENTED** [review]
It's good to make sure the meaning of vars match what they're being used for - if not make another one that's clear.

File: ui/scss/themes/light.scss [hunk 1]
```diff
@@ -171,6 +171,8 @@
   --color-editor-url: var(--color-editor-string);
   --color-editor-hr: var(--color-editor-tag);
   --color-editor-hr-preview: #cccccc;
+  --color-editor-button-hover-bg: #333338;
```
Comment (jessopb) [inline ui/scss/themes/light.scss:174]: The light theme had another lighter color for --color-border

File: ui/scss/component/_markdown-editor.scss [hunk 2]
```diff
@@ -95,7 +95,7 @@
   }
 
   .cm-s-easymde .cm-link {
-    color: var(--color-editor-link);
+    color: var(--color-border);
```
Comment (jessopb) [inline ui/scss/component/_markdown-editor.scss:98]: Another --color-border

File: ui/scss/themes/light.scss (hunk 1 above)
Comment (jessopb) [reply ui/scss/themes/light.scss:174]: Unless screenshots for light theme look really good ?

**Review by jessopb — COMMENTED** [review]
Couple more comments, then if you can paste screenshots of light and dark themes?

Then this should be all set!

File: ui/scss/component/_markdown-editor.scss [hunk 3]
```diff
@@ -114,10 +114,15 @@
       padding-right: 0;
       padding-left: 0;
       border: transparent;
+      margin-right: var(--spacing-xxs);
 
       &:hover {
-        color: var(--color-header-link-active);
+        color: var(--color-header-link);
+        background-color: var(--color-border);
```
[the remark this answers is not in the document]
Comment (jessopb) [reply ui/scss/component/_markdown-editor.scss]: I was unclear here --color-border is different colors on light and dark themes.

**Review by jessopb — APPROVED** [review]
Looks good

**Pull request merged**